exercises/gigasecond/canonical-data.json

```diff
@@ -1,45 +1,58 @@
 {
-   "#": [
-      "The basic test is to add one gigasecond to a few ordinary dates.",
-      "Most test programs currently check only the date and ignore the time.",
-      "For languages with a native date-time type though, expected times",
-      "here show the correct seconds, ignoring leap seconds.",
-      "The date and time formats here are per",
-      "http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.1.15.",
-      "",
-      "Finally, as a demonstration but not really a test,",
-      "some languages demonstrate the add function by inviting the",
-      "solver to include their birthdate in either the solution code",
-      "or test program.  The test program then shows or tests their",
-      "gigasecond anniversay."
-   ],
-   "add": {
-      "description": [
-         "Add one gigasecond to the input."
-      ],
-      "cases": [
-         {
+  "exercise": "gigasecond",
+  "version": "1.0.0",
+  "comments": [
+    "The basic test is to add one gigasecond to a few ordinary dates.",
+    "Most test programs currently check only the date and ignore the time.",
+    "For languages with a native date-time type though, expected times",
+    "here show the correct seconds, ignoring leap seconds.",
+    "The date and time formats here are per",
+    "http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.1.15.",
+    "",
+    "Finally, as a demonstration but not really a test,",
+    "some languages demonstrate the add function by inviting the",
+    "solver to include their birthdate in either the solution code",
+    "or test program.  The test program then shows or tests their",
+    "gigasecond anniversary."
+  ],
+  "cases": [
+    {
+      "add":
+      {
```

Comment:
> It is why we need #658

There is currently no way to ignore certain files there (uses wildcard for all data files). We would have write code for ignoring some files to make build pass for now, and the code will eventually be removed, so I did not do that.